src/ui/qgshananewconnectionbase.ui

```diff
@@ -890,6 +890,16 @@
         </property>
        </widget>
       </item>
+      <item>
+       <widget class="QCheckBox" name="chkUseEstimatedMetadata">
+        <property name="toolTip">
+         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;If checked, estimated table metadata will be used if available. For large tables, this avoids a slow table scans and potentially expensive computations, but may result in incorrect layer properties such as layer extent.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
```

Comment:
What if we rephrase this "this avoids a slow table scans" - > "this avoids slow table loads"?

Reply:
I like that. Done.